grpc/wrappers.go

```diff
@@ -0,0 +1,51 @@
+package grpc
+
+import (
+	"errors"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"github.com/letsencrypt/boulder/publisher"
+	pubPB "github.com/letsencrypt/boulder/publisher/proto"
+)
+
+// PublisherClientWrapper is a wrapper needed to satisfy the interfaces
+// in core/interfaces.go
+type PublisherClientWrapper struct {
+	inner   pubPB.PublisherClient
+	timeout time.Duration
```

Comment:
I'm pretty sure that if you add a new timeout to a context, the first timeout to expire is controlling?